doc/source/georeferencing.md

```diff
@@ -0,0 +1,214 @@
+
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+:mystnb:
+:  code_prompt_show: "Show the code for opening example files"
+:  code_prompt_hide: "Hide the code for opening example files"
+
+import geoutils as gu
+rast = gu.Raster(gu.examples.get_path("exploradores_aster_dem"))
+vect = gu.Vector(gu.examples.get_path("exploradores_rgi_outlines"))
+```
+
+```{code-cell} ipython3
+# Print raster and vector info
+rast.info()
+vect.info()
+```
+
+### Coordinate reference systems
+
+[Coordinate reference systems (CRSs)](https://en.wikipedia.org/wiki/Spatial_reference_system), sometimes also called a 
+spatial reference systems (SRSs), define the 2D projection of the geospatial data. They are stored as a 
+{class}`pyproj.crs.CRS` object in {attr}`~geoutils.Raster.crs`. More information on the manipulation 
+of {class}`pyproj.crs.CRS` objects can be found in [PyProj's documentation](https://pyproj4.github.io/pyproj/stable/).
+
+```{code-cell} ipython3
+# Show CRS attribute of a raster and vector
+print(rast.crs)
+print(vect.crs)
+```
+
+```{note}
+3D CRSs for elevation data are only emerging, and not consistently defined in the metadata.
+The [vertical referencing functionalities of xDEM](https://xdem.readthedocs.io/en/stable/vertical_ref.html) 
+can help define a 3D CRS.
+```
+
+### Bounds
+
+Bounds define the spatial extent of geospatial data, composed of the "left", "right", "bottom" and "top" coordinates.
+The {attr}`~geoutils.Raster.bounds` of a raster or a vector is a {class}`rasterio.coords.BoundingBox` object: 
+
+```{code-cell} ipython3
+# Show bounds attribute of a raster and vector
+print(rast.bounds)
+print(vect.bounds)
+```
+
+```{note}
+To define {attr}`~geoutils.Raster.bounds` consistently between rasters and vectors, {attr}`~geoutils.Vector.bounds`
+ corresponds to {attr}`geopandas.GeoSeries.total_bounds` (total bounds of all geometry features) converted to a {class}`rasterio.coords.BoundingBox`.
+
+To reproduce the behaviour of {attr}`geopandas.GeoSeries.bounds` (per-feature bounds) with a 
+{class}`~geoutils.Vector`, use {attr}`~geoutils.Vector.geom_bounds`.
+```
+
+### Footprints
+
+As reprojections between CRSs deform shapes, including extents, it is often better to consider a vectorized footprint 
+to calculate intersections in different projections. The {class}`~geoutils.Raster.footprint` is a 
+{class}`~geoutils.Vector` object with a single polygon geometry for which points have been densified, allowing 
+reliable computation of extents between CRSs.
+
+```{code-cell} ipython3
+# Print raster footprint
+rast.get_footprint_projected(rast.crs)
+# Plot vector footprint
+vect.get_footprint_projected(vect.crs).plot()
+```
+
+### Grid (only for rasters)
+
+A raster's grid origin and resolution are defined by its geotransform attribute, {attr}`~geoutils.Raster.transform`, and its shape by the data array shape.
+From it are derived the resolution {attr}`~geoutils.Raster.res`, the 2D raster shape 
```

Comment:
It sounds a bit strange. The shape is derived from the shape.
Maybe also mention that `bounds` is also calculated from these two attributes (or at least inter-related) for a raster.

Reply:
Done